package.json

```diff
@@ -50,7 +50,7 @@
     "@folio/stripes-testing": "^4.2.0",
     "@folio/stripes-util": "^5.1.0",
     "@formatjs/cli": "^4.2.20",
-    "axe-core": "^4.3.5",
+    "axe-core": "4.4.3",
```

Comment:
Please file a follow-up ticket to remove this exact-version dep. I totally understand why it's in here now, and I support it, but it's not in our best interest to depend on exact versions in the long term.

Reply:
Good call. Filed https://issues.folio.org/browse/STSMACOM-713 for the upkeep task. eslint-jsx-a11y uses the latest and greatest so we can probably expect some weirdness if that gets hoisted. I'll have to update the version in `stripes-testing` to this version as well...